whylog_vim/tests/tests_output_formater/tests_output_agregator.py

```diff
@@ -0,0 +1,35 @@
+from whylog_vim.output_formater.output_agregator import OutputAgregator
+
+
+def tests_basic_output_agregator():
+    class TestFunctions():
+        def click_function(self, param1, param2):
+            self.test_attr = param1 + param2
```

Comment:
This attribute exist only for one test and only for check function was called. The name addition_attr is not clear for that meaning.